### DISCUSSION TITLE
Failed to set GPU Preference

### DISCUSSION THREAD

**Discussion by jorgerc02** [LizardByte/community#167] [Sunshine] [answered]
I have an AMD CPU and two graphics cards (Radeon Vega Integrated and  GTX1660Ti). When I run the sunshine.exe it uses the Radeon GPU by default so it doesn´t work. I tried setting the 1660Ti as default and disenabling the other one but it gives me another error saying that is has failed to set GPU Preference. Is there any workaround that could fix this?

**Comment by ReenigneArcher** [comment, marked as answer]
Run sunshine as admin or as a service (the default if you use the sunshine installer).

**Comment by locyi** [comment]
Me too, but my cpu is Qualcomm platform,it can't find gpu to encode.
